src/main/java/com/facebook/openwifirrm/ucentral/UCentralClient.java

```diff
@@ -142,6 +142,17 @@ public static void verifySsl(boolean enable) {
 	 */
 	private WebTokenResult accessToken;
 
+	/**
+	 * The unix timestamp (in seconds) keeps track of when the accessToken is created.
+	 */
+	private long created;
```

Comment:
Did you confirm this is needed?? (and shouldn't be fixed in owsec?)

Reply:
Stephan didn't reply to me yet...